code/modules/jobs/job_types/captain.dm

```diff
@@ -96,3 +96,42 @@ Executive Officer
 	head = /obj/item/clothing/head/xocap
 	backpack_contents = list(/obj/item/weapon/storage/box/ids=1,\
 		/obj/item/weapon/melee/classic_baton/telescopic=1, /obj/item/device/modular_computer/tablet/preset/advanced = 1)
+
+/*
+Internal Affairs Agent
+*/
+
+/datum/job/iaa
+	 title = "Internal Affairs Agent"

```

Comment:
The iaa vars are all indented with both a space and a tab. It needs to just be a tab.